### PR TITLE
Added support to copy a file into the container after its creation

### PR DESCRIPTION
**Pull request by Wangor**
Support copy local files to the container after creation

- Changed Setup
- Changed TAR creation
- Added copy functionality after container creation
- Fixed a type in a comment

### CODE REVIEW

File: src/Core/DockerContainerManager.cs [hunk 1]
```diff
@@ -324,6 +323,14 @@ await retryPolicy
                         Instance.Id = response.ID;
                         Instance.Name = startParams.Name;
                     });
+
+                if (_settings.FilesToCopy.Any())
```
Comment (lpzimmermann) [inline src/Core/DockerContainerManager.cs:327]: Why would you need an if around it?

Comment (Wangor) [reply src/Core/DockerContainerManager.cs]: nope... it is resolved

**Comment by fredericbirke** [comment]
Is there an eta when this could be merged? We are currently building an systen integration test system based on squadron and need this as well for copying config files into containers that do not support env vars.

**Review by philbir — APPROVED** [review]

**Comment by philbir** [comment]
We plan to create a release this week.

**Comment by philbir** [comment]
> Is there an eta when this could be merged? We are currently building an systen integration test system based on squadron and need this as well for copying config files into containers that do not support env vars.

We plan to create a release by the end of this week.

**Pull request merged**